myhpom/templates/myhpom/includes/footer.html

```diff
@@ -52,10 +51,17 @@
                             <a class="footer-links-list__link--legal" href="{% url 'myhpom:legal' %}">Legal Disclaimers</a>
                         </li>
                     </ul>
-                    {% scribble 'footer-text' 'shared' %}
-                    <strong>&copy; MIND MY HEALTH</strong>
-                    {% lorem 48 w %}
-                    {% endscribble %}
+                    <div>
+                        <strong>© MIND MY HEALTH</strong> • Mind My Health is a nonprofit project
+                        of The Carolinas Center created with support from The Duke Endowment.
+                        <strong>Mind My Health is NOT a licensed healthcare provider, medical
+                        professional, or law firm.</strong> Please read our complete
+                        <a href="https://www.mindmyhealth.org/legal" alt="Read Terms and Conditions">Terms and Conditions</a>
+                        before using Mind My Health. By using Mind My Health and our services,
+                        you agree to be bound by these Terms. The services are not, and should
+                        never be, a substitute for the advice of a licensed healthcare
+                        provider, medical professional, or attorney.
+                    </div>
```

Comment:
Maybe improve the style of the t&c link? 

<img width="942" alt="screenshot 2018-09-18 10 19 25" src="https://user-images.githubusercontent.com/40370408/45697907-73c44d80-bb2c-11e8-8867-9cc54379161c.png">
<img width="945" alt="screenshot 2018-09-18 10 20 50" src="https://user-images.githubusercontent.com/40370408/45697943-89397780-bb2c-11e8-9158-920e48513362.png">